### PR TITLE
PC-policy-team-build-once || AWS SSM Parameter should be Encrypted

### PR DESCRIPTION
**Pull request by C-Nandhini**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

**PC Policy ID** - 926b7056-6a39-4885-a806-8e4cf958fced

**Compliance Standard** - APRA (CPS 234) Information Security-CPS234-15, APRA (CPS 234) Information Security-CPS234-16, APRA (CPS 234) Information Security-CPS234-17, APRA (CPS 234) Information Security-CPS234-21, APRA (CPS 234) Information Security-CPS234-23, APRA (CPS 234) Information Security-CPS234-27, APRA (CPS 234) Information Security-CPS234-34, Brazilian Data Protection Law (LGPD)-Article 49, CCPA 2018-1798.150(a)(1), CSA CCM v.4.0.1-CEK-03, CSA CCM v.4.0.1-DSP-10, CSA CCM v.4.0.1-UEM-11, Copy of CCPA 2018-1-1798.150(a)(1), Copy of CCPA 2018-1798.150(a)(1), Copy of CCPA 2018555-1798.150(a)(1), Copy2 of CCPA 2018-1798.150(a)(1), CyberSecurity Law of the People's Republic of China-Article 30, CyberSecurity Law of the People's Republic of China-Article 31, CyberSecurity Law of the People's Republic of China-Article 40, Cybersecurity Maturity Model Certification (CMMC) v.1.02-SC.3.177, Cybersecurity Maturity Model Certification (CMMC) v.1.02-SC.3.185, FRB-CCPA 2018-1798.150(a)(1), HITRUST v.9.4.2-Control Reference:06.d, ISO/IEC 27002:2013-10.1.1, ISO/IEC 27002:2013-12.2.1, ISO/IEC 27002:2013-12.3.1, ISO/IEC 27002:2013-13.2.1, ISO/IEC 27002:2013-13.2.3, ISO/IEC 27002:2013-14.1.2, ISO/IEC 27002:2013-14.1.3, ISO/IEC 27002:2013-18.1.3, ISO/IEC 27002:2013-8.3.1, ISO/IEC 27002:2013-8.3.3, ISO/IEC 27017:2015-10.1.1, ISO/IEC 27017:2015-10.1.2, ISO/IEC 27017:2015-6.1.1, ISO/IEC 27018:2019-10.1.2, ISO/IEC 27018:2019-12.3.1, MAS TRM 2021-11.1.1, MAS TRM 2021-11.1.3, MLPS 2.0-8.1.2.2, Marcos Soares test standard-Article 49, NIST 800-53 Rev 5-Protection of Information at Rest | Cryptographic Protection, NIST 800-53 Rev 5-Remote Access | Protection of Confidentiality and Integrity Using Encryption, NIST 800-53 Rev4-AC-17 (2), NIST 800-53 Rev4-SC-28 (1), NIST CSF-PR.DS-1, NIST CSF-PR.DS-5, NIST SP 800-171 Revision 2-3.13.16, NIST SP 800-172-3.4.1e, NIST SP 800-172-3.4.2e, PCI DSS v3.2.1-3.4.1, PIPEDA-4.7.3, Risk Management in Technology (RMiT)-10.51, Risk Management in Technology (RMiT)-10.68, Risk Management in Technology (RMiT)-11.15, Tags including NIST 800-53 Rev 5-Protection of Information at Rest | Cryptographic Protection, Tags including NIST 800-53 Rev 5-Remote Access | Protection of Confidentiality and Integrity Using Encryption

**Remediation Steps:**

1. Sign in to the AWS Console
2. Go to System Manager
3. In the navigation panel, Click on 'Parameter Store'
4. Choose the reported parameter and port it to a new parameter with Type 'SecureString'
5. Delete the reported parameter by clicking on 'Delete'
6. Click on 'Delete parameters'

### CODE REVIEW

File: checkov/terraform/checks/graph_checks/aws/AWSSSMParametershouldbeEncrypted.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+metadata:
+  name: "AWS SSM Parameter should be Encrypted"
+  guidelines: "AWS Systems Manager (SSM) parameters that store sensitive data, for example, passwords, database strings, and permit codes are encrypted so as to meet security and compliance prerequisites. An encrypted SSM parameter is any sensitive information that should be kept and referenced in a protected way."
+  category: ""
+  severity: "medium"
```
Comment (nimrodkor) [inline checkov/terraform/checks/graph_checks/aws/AWSSSMParametershouldbeEncrypted.yaml:5]: @C-Nandhini The `metadata` block should have an ID, following the convention `CKV2_AWS_...`

Comment (C-Nandhini) [reply checkov/terraform/checks/graph_checks/aws/AWSSSMParametershouldbeEncrypted.yaml:5]: @nimrodkor - This policy is part of Prisma and as part of feature parity we need this policy in BC. Can you share me the process to get the ID?

Comment (JamesWoolfenden) [reply checkov/terraform/checks/graph_checks/aws/AWSSSMParametershouldbeEncrypted.yaml:5]: Search the codebase for CKW_AWS_ - take the next number - currently ~199

Comment (nimrodkor) [reply checkov/terraform/checks/graph_checks/aws/AWSSSMParametershouldbeEncrypted.yaml:5]: This is a graph check, so the next ID should be `CKV2_AWS_34` @C-Nandhini

Comment (C-Nandhini) [reply checkov/terraform/checks/graph_checks/aws/AWSSSMParametershouldbeEncrypted.yaml]: @nimrodkor - ID is added, please check.

**Comment by nimrodkor** [comment]
I'll update the category after merging this PR

**Pull request merged**